packages/docs/package.json

```diff
@@ -21,8 +21,9 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "tslint src/**/**.ts?",
+    "lint": "tslint src/**/**.tsx?",
```

Comment:
aqui pode ser `tslint 'src/**/*.{ts,tsx}'`, pra manter o padrão =)

Reply:
mas cara, é a mesmíssima coisa, a interrogação torna o x opcional. E se for falar de padrão, tsconfig do docs e do web tao assim hehe

Mas enfim, vou mudar, esse pr ta aberto faz tempo ja